packages/context-entries/lib/Keys.ts

```diff
@@ -173,6 +173,7 @@ export const KeysInitQuery = {
    * A JSON-LD context
    */
   jsonLdContext: new ActionContextKey<any>('@context'),
+  disableCaching: new ActionContextKey<boolean>('@comunica/actor-init-query:disableCaching'),
```

Comment:
```suggestion
  noCache: new ActionContextKey<boolean>('@comunica/actor-init-query:noCache'),
```

Could you also document this briefly?